Formula/supervisor.rb

```diff
@@ -5,6 +5,7 @@ class Supervisor < Formula
   homepage "http://supervisord.org/"
   url "https://github.com/Supervisor/supervisor/archive/4.2.0.tar.gz"
   sha256 "05031f36ad15cad47fb56f01d8e075f952ae39ba8ce492ea790ebb310e3f0368"
+  revision 1
```

Comment:
Doesn't feel like this needs a revision?

Reply:
The sample config file (which is embedded in the bottle) has been renamed. Doesn't that require building a new bottle?

Reply:
`revision` is not required to build a new bottle.

Reply:
Ah OK. Will remove that then.